.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@ https://github.com/blog/1506-closing-issues-via-pull-requests for more
 information.
 
 ### checklist
-- [ ] Make sure you read and understood the [CONTRIBUTING.md](https://github.com/xh3b4sd/anna/.github/CONTRIBUTING.md).
+- [ ] Make sure you read and understood the [CONTRIBUTING.md](/.github/CONTRIBUTING.md).
```

Comment:
I too would prefer a relative link, but as it seems to be handled in different ways across different pages on GitHub, an absolute URL is probably the way to go.